### ISSUE TITLE
Automatically set timezone

### ISSUE THREAD

**Issue by mrbluecoat** [MichaIng/DietPi#5620]
#### Is your feature request related to a problem? Please describe:
This is a low priority enhancement since dietpi.txt has `AUTO_SETUP_TIMEZONE` but https://github.com/mrbluecoat/tzme might be a nice default install for your image to allow for automatic detection of the timezone.

**Comment by MichaIng** [comment]
Many thanks for your request

We could use our own service at `dietpi.com/geoip` for this, although it would require some mapping and doesn't work with countries with multiple time zones easily. Probably we can make the time zone info available explicitly on our own domain. We made quite bad experience with external GeoIP services being blocked by many block lists and DNS providers, so I'm not keen to implement a feature based on such again 😉.
